### PR TITLE
ci: run tests in other platforms (windows, wasm)

### PR DESCRIPTION
**Pull request by morenol**
## Change Summary
This adds to the CI, a worker to run the tests with Windows and also using wasm-pack for WASM.

## PR Checklist

- [x] I have read and signed the [Contributor License Agreement](https://forms.gle/PZyiY5N2GDQU8GsV9).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,15 +30,23 @@ jobs:
...
+          args: --all-features --target ${{ matrix.platform.target }}
       - name: Tests
+        if: matrix.platform.target != 'wasm32-unknown-unknown'
         uses: actions-rs/cargo@v1
         with:
           command: test
           args: --all-features
+      - name: Tests in WASM
+        if: matrix.platform.target == 'wasm32-unknown-unknown'
+        run: wasm-pack test --node -- --all-features
```
Comment (morenol) [inline .github/workflows/ci.yml:52]: Probably we need to change this to --features wasm, in the PR with the transport implementation

Comment (emmanuelantony2000) [reply .github/workflows/ci.yml:52]: Will do that.

**Review by emmanuelantony2000 — COMMENTED** [review]
To solve the problem of using HTTPMock which might not run in Wasm, I am trying to use it in the [standalone](https://docs.rs/httpmock/0.5.8/httpmock/#standalone-mode) mode. We could include that maybe later (not in this PR), if it proves to be working.

**Comment by morenol** [comment]
Fine! Can we merge this now?

**Comment by emmanuelantony2000** [comment]
Sure

**Pull request merged**